test/crypten/test_context.py

```diff
@@ -26,14 +26,14 @@ def forward(self, x):
         out = self.conv1(x)
         out = F.relu(out)
         out = F.max_pool2d(out, 2)
-        out = out.view(out.size(0), -1)
+        out = out.view(-1, 16 * 12 * 12)
```

Comment:
This was changed because of [this](https://github.com/facebookresearch/CrypTen/pull/121)